components/producers/golang-nancy/main_test.go

```diff
@@ -145,3 +146,16 @@ var exampleOutput = `{
 	]
   }
 `
+
+func TestGetCWEFromTitle(t *testing.T) {
+	cwe := getCWEFromTitle("[CVE-2023-26125] CWE-20: Improper Input Validation")
+	assert.Equal(t, "CWE-20", cwe)
```

Comment:
we should `require` instead of `assert`. If there is an error `assert` wont' stop the test, also this should be implemented as a table driven test